v19.2/start-a-local-cluster-in-docker-windows.md

```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
 
-    ~~~ powershell
-    PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
-    ~~~
+   ```powershell
+   PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
+   ```
 
-    You'll see the following message:
+   You'll see the following message:
 
-    ~~~
-    Cluster successfully initialized
-    ~~~
+   ```
+   Cluster successfully initialized
+   ```
 
 ## Step 3. Use the built-in SQL client
 
 Now that your cluster is live, you can use any node as a SQL gateway. To test this out, let's use the `docker exec` command to start the [built-in SQL shell](cockroach-sql.html) in the first container.
 
 1. Start the SQL shell in the first container:
 
-    ~~~ powershell
-    PS C:\Users\username> docker exec -it roach1 ./cockroach sql --insecure
-    ~~~
+   ```powershell
+   PS C:\Users\username> docker exec -it roach1 ./cockroach sql --insecure
+   ```
 
 2. Run some basic [CockroachDB SQL statements](learn-cockroachdb-sql.html):
 
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > CREATE DATABASE bank;
-    ~~~
-
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > CREATE TABLE bank.accounts (id INT PRIMARY KEY, balance DECIMAL);
-    ~~~
-
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > INSERT INTO bank.accounts VALUES (1, 1000.50);
-    ~~~
-
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > SELECT * FROM bank.accounts;
-    ~~~
-
-    ~~~
-      id | balance
-    +----+---------+
-       1 | 1000.50
-    (1 row)
-    ~~~
+   {% include copy-clipboard.html %}
+
+   ```sql
+   > CREATE DATABASE bank;
+   ```
+
+   {% include copy-clipboard.html %}
+
+   ```sql
+   > CREATE TABLE bank.accounts (id INT PRIMARY KEY, balance DECIMAL);
+   ```
+
+   {% include copy-clipboard.html %}
+
+   ```sql
+   > INSERT INTO bank.accounts VALUES (1, 1000.50);
+   ```
+
+   {% include copy-clipboard.html %}
+
+   ```sql
+   > SELECT * FROM bank.accounts;
+   ```
+
+   ```
+     id | balance
+   +----+---------+
+      1 | 1000.50
+   (1 row)
+   ```
 
 3. Now exit the SQL shell on node 1 and open a new shell on node 2:
 
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > \q
-    ~~~
+   {% include copy-clipboard.html %}
 
-    ~~~ powershell
-    PS C:\Users\username> docker exec -it roach2 ./cockroach sql --insecure
-    ~~~
+   ```sql
+   > \q
+   ```
+
+   ```powershell
+   PS C:\Users\username> docker exec -it roach2 ./cockroach sql --insecure
+   ```
 
 4. Run the same `SELECT` query as before:
 
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > SELECT * FROM bank.accounts;
-    ~~~
+   {% include copy-clipboard.html %}
+
+   ```sql
+   > SELECT * FROM bank.accounts;
+   ```
 
-    ~~~
-      id | balance
-    +----+---------+
-       1 | 1000.50
-    (1 row)
-    ~~~
+   ```
+     id | balance
+   +----+---------+
+      1 | 1000.50
+   (1 row)
+   ```
 
-    As you can see, node 1 and node 2 behaved identically as SQL gateways.
```

Comment:
This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.